noxfile.py

```diff
@@ -104,7 +104,7 @@ def audit(session, installable):
     Audit dependencies for vulnerabilities.
     """
     session.install("pip-audit", installable)
-    session.run("python", "-m", "pip_audit")
+    session.run("python", "-m", "pip_audit", "-r", str(REQUIREMENTS["docs"]))
```

Comment:
Thanks, yeah this is related to https://github.com/crate-py/rpds/pull/178#issuecomment-3366748279 -- here I think the fix is different a bit from just this.

This noxenv is meant to ensure "things jsonschema depends on do not have open CVEs." -- for that we need to run it not just on the docs requirements file but on each set of dependencies (i.e. each possible way of installing jsonschema), which is what `installable` represents here (basically it's a packaging extra).

So this isn't exactly right as-is, but my confusion from the comment I linked still leads me to not know what the right fix is here yet, as indeed `pip` *isn't* a dependency.

Reply:
Yeah this commit feels very "hacky" and not a permanent solution. I'm unsure how to fix it though. Should I drop this commit?

Reply:
Yes maybe that's best and I'll see if I can understand better what the intended fix really is from pip-audit.

Reply:
Removed 👍